codeworld-api/src/CodeWorld/Picture.hs

```diff
@@ -53,10 +53,23 @@ data Picture
     | SolidClosedCurve CallStack
               [Point]
     | Path CallStack
+              [Point]
+    | Polygon CallStack
+           [Point]
+    | ThickPolygon CallStack
+           [Point]
```

Comment:
You still need a width for thickPolygon.  Same for all the other Thick variants.

Reply:
this seems to work (dunno if I'm doing it right), but what do I do with the picObj that has w in scope in the return type but doesn't actually use w? I'll push up the code also as I have it.

Reply:
The width can be set to zero for shapes that are not thick.  That will do the same thing it did before.